packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/__tests__/accumulator.spec.tsx

```diff
@@ -23,13 +27,19 @@ describe('Accumulator', () => {
         expect(screen.getByText('3%')).toBeInTheDocument();
         expect(screen.getByText('4%')).toBeInTheDocument();
         expect(screen.getByText('5%')).toBeInTheDocument();
-        expect(screen.getByText('1%').getAttribute('class')).toContain('--selected');
+        expect(screen.getByText('1%')).toHaveClass('number-selector__selection--selected');
     });
 
     it('3% growth_rate should be selected when 0.03 is a currently selected and stored growth_rate value', () => {
         mock_connect_props.growth_rate = 0.03;
         render(<Accumulator />);
-        expect(screen.getByText('3%').getAttribute('class')).toContain('--selected');
-        expect(screen.getByText('1%').getAttribute('class')).not.toContain('--selected');
+        expect(screen.getByText('3%')).toHaveClass('number-selector__selection--selected');
+        expect(screen.getByText('1%').getAttribute('class')).not.toContain('number-selector__selection--selected');
```

Comment:
Is it possible not to use `.getAttribute` here as we tried to avoid using such methods?
Maybe `.not.toHaveClass`

Reply:
yes. Will change.